FORMAT.md

```diff
@@ -154,15 +161,14 @@ The rigid body refers to a solid body with mass and one or more outputs. Default
 - `output_rot` (rotation matrix): The default orientation of the output frames. Defaults to an identity matrix.
 - `output_trans` (translation vector, m): The default position the output frames. Defaults to (0,0,0).
 - `ixx`, `iyy`, `izz`, `ixy`, `ixz`, `iyz` (floating point formulae, kg m^2): The 6 elements of the inertia tensor, relative to the COM frame as given above.  Each defaults to 0 (note, this means overall default is a point mass).
-- `mesh_path` (string): Relative path to a file used to store 3D mesh information for visualization purposes. A forward slash should be used as a file separation character. Paths are relative to the current HRDF file being parsed; absolute paths are not allowed.  The double dot ".." pattern moves up a directory.  Supported file types and extensions depend on the application consuming the HRDF file for visualization.
+- `mesh_path` (string): Relative file path or web URL to a file used to store 3D mesh information for visualization purposes. A forward slash should be used as a file separation character. File paths are relative to the current HRDF file being parsed; absolute paths are not allowed.  The double dot ".." pattern moves up a directory.  Web URLs must start with `http://` or `https://`. Supported file types, extensions, and sources (e.g., local vs web) depend on the application consuming the HRDF file for visualization.
```

Comment:
I was originally thinking of a CSS like `url(...)` scheme, but I don't see a point over just using `http(s)://`. One thing to figure out is handling relative files, e.g., `.obj` files internally link to material files for coloring.